library/Fission/CLI/Environment/Override/Types.hs

```diff
@@ -48,7 +47,7 @@ instance Monoid Override where
 
 instance ToJSON Override where
   toJSON Override {..} = object $ catMaybes
-    [ ("peers"      .=) <$> tailMaybe peers
+    [ ("peers"      .=) <$> pure peers
```

Comment:
Oh lol yeah that would do it

Reply:
This looks fairly intentional. I wonder why we had it like that? Just a quick “the server is updated and we’ll do the CLI at some stage”?

Reply:
right I noted that it looks intentional as well. But I can't figure out why we'd do it either....

> the server is updated and we’ll do the CLI at some stage

that seems most likely, but even so you'd think we'd do `head` instead of `tail`

Reply:
Well, looks much more correct now 👍